coda-prototype/src/model.ts

```diff
@@ -1394,12 +1393,14 @@ class Code {
         this._words = [];
         this._isEdited = isEdited;
         this._eventsWithCode = new Map();
-        if (regex && regex[0] && regex[0].length > 0) {
+
+        if (regex && regex instanceof Array && regex.length === 2 && regex[0] && regex[0].length > 0) {
```

Comment:
if this clause isn't true, what does that mean? Why should that happen?

Reply:
It means the regex passed to Code constructor is not actually of type [string, string].

The previous version of Coda saved regexes of type string to internal storage. If users migrate to this version with a regex in internal storage, this will break without these checks. Previously, regexes were not loading from storage at all - I choose to overwrite : string regexes with ["", ""], which essentially gives the same behaviour (from users' perspectives) as the previous version, but only until they save a new regex.